css/fonts.css

```diff
@@ -0,0 +1,43 @@
+@font-face {
+  font-family: "Montserrat";
+  src: 
+    url("../fonts/MontserratAlternates-Bold.eot"),
+    url("../fonts/MontserratAlternates-Bold.eot?#iefix") format("embedded-opentype"),
+    url("../fonts/MontserratAlternates-Bold.woff") format("woff"),
+    url("../fonts/MontserratAlternates-Bold.woff2") format("woff2");
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Montserrat";
+  src:
+    url("../fonts/MontserratAlternates-Medium.eot"),
+    url("../fonts/MontserratAlternates-Medium.eot?#iefix") format("embedded-opentype"),
+    url("../fonts/MontserratAlternates-Medium.woff") format("woff"),
+    url("../fonts/MontserratAlternates-Medium.woff2") format("woff2");
+  font-weight: 500;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Montserrat";
+  src: 
+    url("../fonts/MontserratAlternates-Regular.eot"),
+    url("../fonts/MontserratAlternates-Regular.eot?#iefix") format("embedded-opentype"),
+    url("../fonts/MontserratAlternates-Regular.woff") format("woff"),
+    url("../fonts/MontserratAlternates-Regular.woff2") format("woff2");
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Montserrat";
+  src: 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5516260)